src/app.ts

```diff
@@ -22,30 +24,27 @@ const env = nunjucks.configure([
   express: app,
 });
 
-// TODO Configure the view engine for CHS - uncomment the following lines and change as appropriate
-
 env.addGlobal("CDN_URL", process.env.CDN_HOST);
 
 // env.addGlobal("ERROR_SUMMARY_TITLE", ERROR_SUMMARY_TITLE);
 env.addGlobal("PIWIK_URL", PIWIK_URL);
 env.addGlobal("PIWIK_SITE_ID", PIWIK_SITE_ID);
 
-// app.enable("trust proxy");
-// app.use(express.json());
-// app.use(express.urlencoded({ extended: false }));
-// app.use(cookieParser());
-// app.use(sessionMiddleware);
-// app.use(`${pageURLs.EXTENSIONS}/*`, authenticate);
+app.enable("trust proxy");
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(sessionMiddleware);
+
+app.use(`${pageURLs.PROMISE_TO_FILE}/*`, authenticate);
 
 // view engine setup
 app.set("views", path.join(__dirname, "views"));
 app.set("view engine", "html");
 
-app.use(logger("dev"));
-app.use(express.json());
```

Comment:
These 4 lines just needed to move up, above the authentication redirection config, for sign in to work.